lib/cc/engine/analyzers/violation.rb

```diff
@@ -57,7 +61,15 @@ def name
         end
 
         def calculate_points
-          base_points * issue.mass
+          if issue.mass > threshold
+            base_points + overage * points_per
```

Comment:
Thought about that. Was on fence but now settled and agree.

Making change - thanks!